public/squawk.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+  <link rel="stylesheet" href="squawk.css">
+  <script src="/squawk.js"></script> 
```

Comment:
[suggestion] Rather than having both the host and the de-ref HTML call the same JS and just do a URL check, have a second piece of code here that just redirects this page immediately to a random IP.